com.unity.mobile.notifications/Runtime/Android/Plugins/com/unity/androidnotifications/UnityNotificationUtilities.java

```diff
@@ -418,7 +426,7 @@ private static String deserializeString(DataInputStream in) throws IOException {
                 return b.getParcelable("obj");
             }
         } catch (Exception e) {
-            Log.e("Unity", "Failed to deserialize parcelable", e);
+            Log.e("UnityNotifications", "Failed to deserialize parcelable", e);
```

Comment:
Please extract "UnityNotifications" into a constant and reuse it, since it's duplicating everywhere